README.md

```diff
@@ -175,6 +183,39 @@ New-PowerAppManagementApp -ApplicationId $appId # Must be run from a Power Platf
 
 > **Notes**: Only authentication via `CertificateThumbprint` is currently supported. We will also be supporting automated app registration in a later release.
 
+### AAD Conditional Access Policy Exemptions
+The ScubaGear ConfigFilePath command line option allows users to define custom variables in YAML format for use in policy assessments against the AAD baseline.  These custom variables are used to exempt specific user and group exclusions from conditional access policy checks that normally would not pass if exclusions are present.  These parameters support operational use cases for having backup or break glass account exclusions to global user policies without failing best practices.  Any exemptions and their risks should be carefully considered and documented as part of an organization's cybersecurity risk management program process and practices.
+
+**YAML AAD Configuration File Syntax and Examples**
+
+**Aad** refers to the AAD specific variables that enable users to specify user, group, and role exclusions that are documented exemptions to select conditional access policies in the AAD configuration policy baselines. Users, groups, and roles are specified by their respective Universally Unique Identifier (UUID) in the tenant. This variable set is only needed if the agency has documented exemptions for one or more of the conditional access policy baselines:
+- MS.AAD.1.1: *Legacy authentication SHALL be blocked*
+- MS.AAD.2.1: *Users detected as high risk SHALL be blocked*
+- MS.AAD.2.3: *Sign-ins detected as high risk SHALL be blocked*
+- MS.AAD.3.2: *Phishing-resistant MFA SHALL be enforced for all users*
+- MS.AAD.3.3: *If phishing-resistant MFA has not been enforced yet, then an alternative MFA method SHALL be enforced for all users*
+- MS.AAD.3.7: *Managed devices SHOULD be required for authentication*
+- MS.AAD.3.8: *Managed Devices SHOULD be required to register MFA*
+- MS.AAD.7.4: *Permanent active role assignments SHALL NOT be allowed for highly privileged roles*
+
+The example below illustrates the syntax for defining user, group, and role exemptions to select policies. It also makes use of a YAML anchor and alias to simplify formatting policies having the same documented exemptions. Items surrounded by chevrons are to be supplied by the user.
```

Comment:
As a user, it is not clear to me when I can use a CapExclusion and/or a RoleExclusion.

Reply:
added some text that tries to clarify that..